### PR TITLE
Add about courses page

### PR DESCRIPTION
**Pull request by eleanorreem**
Screenshot 2022-04-21 at 11 30 53

### CODE REVIEW

File: pages/courses/about.tsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    version: preview ? 'draft' : 'published',
+    cv: preview ? Date.now() : 0,
+    language: locale,
+  };
+
+  let { data } = await Storyblok.get(`cdn/stories/courses/about`, sbParams);
+
+  return {
+    props: {
+      story: data ? data.story : null,
```
Comment (swetha-chayn) [inline pages/courses/about.tsx]: I think if you rename this `story` key here to `storyProps` then it'll be passed in with that name into the class.

File: pages/courses/about.tsx [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+interface Props {
+  story: StoryData;
+  preview: boolean;
+  sbParams: StoriesParams;
+  locale: LANGUAGES;
+}
+
+const CourseAbout: NextPage<Props> = ({ story: storyProps, preview, sbParams, locale }) => {
+  const story = useStoryblok(storyProps, preview, sbParams, locale);
```
Comment (swetha-chayn) [inline pages/courses/about.tsx:21]: I like the naming `storyProps` and `story` to separate the two variables 👍 

Shall we do that going forward?

**Review by swetha-chayn — APPROVED** [review]
Looks good 👍 

Left some comments - up to you to.

File: pages/courses/about.tsx (hunk 2 above)
Comment (eleanorreem) [reply pages/courses/about.tsx:21]: 👍  i think reassigning is a bit rogue

**Pull request merged**